src/test/java/jenkins/plugins/logstash/LogstashConsoloLogFilterTest.java

```diff
@@ -122,4 +136,21 @@ public void decorateLoggerSuccessBadWriter() throws Exception {
     assertEquals("Error was not written", "Mocked Constructor failure", buffer.toString());
     verify(mockWriter).isConnectionBroken();
   }
+
+  @Test
+  public void decorateLoggerSuccessEnabledGlobally() throws IOException, InterruptedException
```

Comment:
can you also add a negative case (won't send if not enabled globally nor per-job)?

Reply:
the method decorateLoggerSuccessNoLogstashBuildWrapper is this test, will rename this method to make it more clear.

Reply:
right, I forgot the existing negative tests should cover it